src/pages/coalition-partners.tsx

```diff
@@ -6,28 +6,37 @@ import {Heading,
   WrapItem,
 } from "@chakra-ui/react"
 import { Layout } from '../Components'
+import TechActionImage from '../../static/Images/Partners/NYC_DSA_TechAction.jpg'
+import IBEWImage from '../../static/Images/Partners/IBEW.jpg'
+import VocalNYImage from '../../static/Images/Partners/VocalNY.jpg'
+import MOREImage from '../../static/Images/Partners/MORE.jpg'
```

Comment:
According to [the Gatsby docs](https://www.gatsbyjs.com/docs/how-to/images-and-media/importing-assets-into-files/), importing them like this instead of using the static folder path as the `src` is more correct.